src/main/resources/tasks/cvpartner-import/cvpartner-import.ts

```diff
@@ -29,6 +39,7 @@ export function run(): void {
 
   const [changed, unchanged] = cvPartnerEmployees.reduce((updateResult, employee, index) => {
     const cvPartnerProfile = fetchEmployeeProfile(employee.id, employee.default_cv_id);
+    rateLimit();
```

Comment:
Hadde det gitt mer mening å ha denne på slutten av metoden isteden? for da kan den behandle dataene som alt er hentet, så ta pausen? I guess det egentlig ikke betyr så mye

Reply:
Eventuelt først?